scipy/signal/__init__.py

```diff
@@ -64,6 +64,9 @@
    hilbert       -- Compute 1-D analytic signal, using the Hilbert transform.
    hilbert2      -- Compute 2-D analytic signal, using the Hilbert transform.
 
+   iirnotch      -- Design second-order IIR notch digital filter.
```

Comment:
It should go to "Filter Design" section below or to "Matlab-style IIR filter design".